src/components/Homepage/CommunicateThroughSection.jsx

```diff
@@ -1,8 +1,7 @@
 import { withTranslation } from "next-i18next";
 
-import PageIntro from "@/components/PageIntro";
-
 import CommunicateThroughItem from "./CommunicateThroughItem";
+import PageIntro from "../PageIntro";
```

Comment:
- It will be better to use `Absolute Import`

Reply:
@dhiashalabi No it's NOT, 
since in the configuration 
![image](https://user-images.githubusercontent.com/96573043/231114894-6a73ef3a-d5c2-487d-8476-93f0e16040dc.png)
(components) directory is not aliasing to use as absolute import.
+ this issue is only concerned with rearranging the homepage components in another subdirectory, not modifying the configration(jsconfig.json)